datamade/committee-oversight
# CHP rating fixes

## Overview

Closes #118. This PR makes two fixes to the CHP rating calculation:

1. Corrects a typo in the list of category types to include in the `total_hearings` counts
2. Adds handling for updating committee ratings that already exist (previously, the import script would import another `CommitteeRating` with duplicate committee and Congress values instead of updating)

Jamie confirmed that these numbers are looking right now! 

## Testing Instructions

* Make sure you have the `hearings.dump` described in the testing instructions [here](https://github.com/datamade/committee-oversight/pull/117#issue-340155447).
* `docker-compose down --volumes`
* Follow instructions in README to run site
* Visit http://localhost:8000/committee-61caba6d-71bc-41c7-882e-32ec97768f63/ and confirm you see these numbers:

![Screen Shot 2019-11-25 at 1 04 23 PM](https://user-images.githubusercontent.com/1094243/69569872-2fc8a600-0f84-11ea-8465-f8da3eb1e8b4.png)
